### DISCUSSION TITLE
스크립트에 public으로 노출시켜 추가하는 인자가 사라지는 현상

### DISCUSSION THREAD

**Discussion by mzdokebi** [naverz/zepeto-studio-kor#908] [5. Q&A]
![spawnerror01](https://user-images.githubusercontent.com/103915942/174911367-cb181f16-09b3-4d6a-8e08-4197c7536f68.png)
![spawnerror02](https://user-images.githubusercontent.com/103915942/174911413-9ee28ac1-a9f9-4558-b89e-785e621d455e.png)
![spawnerror03](https://user-images.githubusercontent.com/103915942/174911419-f71707b4-9bc5-47be-9f62-1f5a2399b975.png)

Respawn 스크립트에 추가한 respawnPose 오브젝트가 실행시 사라짐.
스크립트는 트리거 발생시 respawnPose로 텔레포트하는 것 말고는 없음.
인자로 추가하는 오브젝트가 사라지는 현상은 이것 말고도 너무 자주 발생해서 상당히 곤란함.
이름으로 검색해서 해결은 하고 있지만 그래픽과의 협업시 특정한 이름을 유지해야 하기 때문에 버그 발생 위험이 높음.

**Comment by hanaaple** [comment]
저도 겪었던 오류인데 실행 시 프리팹에 달려있는 제페토 스크립트 변수가 씬 기준이 아닌 프리팹 기준으로 바뀝니다.
언팩을 해주거나 프리팹 자체에 변수를 넣어주는 방법을 사용해보세요
